modules/hcp-ec2-client/variables.tf

```diff
@@ -38,3 +38,9 @@ variable "consul_version" {
 variable "security_group_id" {
   type = string
 }
+
+variable "vpc_cidr" {
+  type        = string
+  description = "VPC CIDR"
+  default     = "10.0.0.0/8"
+}
```

Comment:
Why did you chose to make this configurable for this example? We try to keep our examples similar across the board (ec2, ecs, eks). If there is something we would want to add to ec2, we might want to make that change for the others as well.

Reply:
When using this module, a consumer could have a different VPC CIDR range. We had it hard coded to "10.0.0.0/8" which is used in the consul's client.json configuration for `bind_addr`. 

I was using this module with a VPC CIDR of 172.16.0.0/31 and was trying to work out why consul was failing to start. 

Existing demos and environments using this module should still work but anyone with a different VPC has the ability to change the CIDR range for their environment.